src/test/java/jp/co/yahoo/yosegi/spread/column/TestPrimitiveCellManager.java

```diff
@@ -0,0 +1,346 @@
+  }
+
+  @Test
+  public void T_filter_2() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test" ) , 1 );
+    cellManager.add( new StringObj( "test" ) , 3 );
+    cellManager.add( new StringObj( "test" ) , 5 );
+
+    boolean[] filters = new boolean[cellManager.size()];
+    assertEquals( filters.length , 6 );
+
+    assertEquals( filters[0] , false );
+    assertEquals( filters[1] , false );
+    assertEquals( filters[2] , false );
+    assertEquals( filters[3] , false );
+    assertEquals( filters[4] , false );
+    assertEquals( filters[5] , false );
+    cellManager.filter( new NotNullFilter( ColumnType.STRING ) , filters );
+    assertEquals( filters[0] , false );
+    assertEquals( filters[1] , true );
+    assertEquals( filters[2] , false );
+    assertEquals( filters[3] , true );
+    assertEquals( filters[4] , false );
+    assertEquals( filters[5] , true );
+  }
+
+  @Test
+  public void T_getPrimitiveObjectArray_1() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test1" ) , 1 );
+    cellManager.add( new StringObj( "test3" ) , 3 );
+    cellManager.add( new StringObj( "test5" ) , 5 );
+
+    PrimitiveObject[] result = cellManager.getPrimitiveObjectArray(
+        new AllExpressionIndex( cellManager.size() ) , 0 , 10 );
+
+    assertEquals( result[0] , null );
+    assertEquals( result[1].getString() , "test1" );
+    assertEquals( result[2] , null );
+    assertEquals( result[3].getString() , "test3" );
+    assertEquals( result[4] , null );
+    assertEquals( result[5].getString() , "test5" );
+    assertEquals( result[6] , null );
+    assertEquals( result[7] , null );
+    assertEquals( result[8] , null );
+    assertEquals( result[9] , null );
+  }
+
+  @Test
+  public void T_getPrimitiveObjectArray_2() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test1" ) , 1 );
+    cellManager.add( new StringObj( "test3" ) , 3 );
+    cellManager.add( new StringObj( "test5" ) , 5 );
+
+    PrimitiveObject[] result1 = cellManager.getPrimitiveObjectArray(
+        new AllExpressionIndex( cellManager.size() ) , 0 , 3 );
+
+    assertEquals( result1[0] , null );
+    assertEquals( result1[1].getString() , "test1" );
+    assertEquals( result1[2] , null );
+
+    PrimitiveObject[] result2 = cellManager.getPrimitiveObjectArray(
+        new AllExpressionIndex( cellManager.size() ) , 3 , 3 );
+
+    assertEquals( result2[0].getString() , "test3" );
+    assertEquals( result2[1] , null );
+    assertEquals( result2[2].getString() , "test5" );
+  }
+
```

Comment:
Please describe what are tested in the test method separately at T_getPrimitiveObjectArray_?.
For example, T_getPrimitiveObjectArray_when_xxx or some.

Reply:
Please add a prefix or suffix like T_ or Text or some so that the code reader knows it is a test method including assertThat or assertThrows and/or some testing methods.